docs/how-it-works.mdx

```diff
@@ -0,0 +1,116 @@
+---
+sidebar_position: 2
+---
+
+# How it Works
+
+Learn the high-level workings of Divvi Up and how it accomplishes the
+privacy-preserving property.
+
+## Breakdown
+
+Divvi Up functionality can be broadly broken down into five steps.
+
+See the [Distributed Aggregation Protocol (DAP)][DAP] for a highly detailed
+specification of the protocol that underpins Divvi Up.
+
+<!-- TODO(inahga): Consider making this a more detailed `mermaid` diagram -->
+
+![](../static/how-it-works.png#gh-dark-mode-only)
+![](../static/how-it-works-light.jpg#gh-light-mode-only)
+
+### Step 1: Client generate measurements
+
+An app developer wants to collect statistics about their app, such as user
+behavior, demographics, or system information. The developer writes app
+functionality that takes measurements of these properties.
+
+In Divvi Up, the developer creates a Task, which encodes data about the
```

Comment:
This is included in step 1 but feels like step 0, as it happens before `Client generates measurements`

Reply:
Let me see how a step 0 looks--this might also be a handy place to include a quick blurb on the state of onboarding (i.e. get invited by existing user or ask us).